app/assets/javascripts/components/StickyColumn.js

```diff
@@ -0,0 +1,215 @@
+define(['jquery', 'DoughBaseComponent', 'eventsWithPromises'], function($, DoughBaseComponent, eventsWithPromises) {
+  'use strict';
+
+  var StickyColumn,
+      defaultConfig = {},
+      $parent,
+      contentHeight,
+      topMargin,
+      top,
+      bottom,
+      isFixed,
+      isAtBottom,
+      isInSidebar,
```

Comment:
The variables above might be better moved into the `StickyColumn` *class*, because it will give you a little more control if you want to subclass this in the future.